start_firebase_listener.sh

```diff
@@ -40,6 +40,8 @@ fi
 timeout_cmd "[ $(curl --write-out %{http_code} --silent --output /dev/null $ES_HOST:9200) -eq 200 ]"
 
 #start the app
-node app.js &
-cd example
-serve
+node app.js
```

Comment:
Didn't notice this yesterday, but it's probably a good idea to prefix this command with exec (http://wiki.bash-hackers.org/commands/builtin/exec)